SMRefLocation/examples/example.json

```diff
@@ -1,14 +1,16 @@
 {
   "id": "urn:ngsi-ld:RefLocation:00",
   "type": "SMRefLocation",
-  "locationName": "Thessaloniki",
+  "locationName": {
+    "type": "string",
+    "streetAddress": "Thessaloniki"
+  },

```

Comment:
for a key-values example this is wrong, this should use address object and addressLocality instead

Reply:
Can you explain further? I am referencing address object already, I will just change streetAddress to addressLocality name, right?

Reply:
It would be helpful if I could have the correct writing, if my .json is wrong again.